__mocks__/zustand.ts

```diff
@@ -0,0 +1,44 @@
+// https://docs.pmnd.rs/zustand/guides/testing
```

Comment:
Må denne fila ligge under `__mocks__`? Eller kan den ligge inni den andre mock-mappa, så får vi én mappe mindre i prosjektet?

Reply:
Den må ligge der for at automagien skal fungere, men kan sikkert peke på den fra en config.